runners/flink/runner/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/FlinkGroupByKeyWrapper.java

```diff
@@ -64,10 +66,12 @@ public K getKey(WindowedValue<KV<K, V>> value) throws Exception {
         });
   }
 
-  // special type to return as key for null key
+  /**
+   * Special type to return  as key for null key.
+   */
   public static class VoidValue {
     private VoidValue() {}
 
-    public static VoidValue INSTANCE = new VoidValue();
+    public static VoidValue instance = new VoidValue();
```

Comment:
I think the style guide says that only constants are capitalized, so `static final VoidValue INSTANCE`. It seems like making this final might be reasonable?